server/routes/auth/providers/google.ts

```diff
@@ -112,6 +117,17 @@ if (env.GOOGLE_CLIENT_ID && env.GOOGLE_CLIENT_SECRET) {
             }
 
             if (!team) {
+              // No team usually means this is the apex domain
+              // Throw different errors depending on whether we think the user is
+              // trying to create a new account, or log-in to an existing one
+              const userExists = await User.count({
+                where: { email: profile.email.toLowerCase() },
```

Comment:
is this purposefully a global find without `teamId`?

Reply:
yeah, it's to determine if this is a new email altogether — the case that it's nested inside is most likely coming from the apex domain